samples/http_redirect/plugin.cc

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <boost/url/parse.hpp>
```

Comment:
Can you add tags so the recipes are embeddable in docs? (see other samples)

Basically wrap both plugins in:
// [START serviceextensions_http_redirect]
// [END serviceextensions_http_redirect]